www/profile/inner.js

```diff
@@ -437,7 +437,7 @@ define([
         if (APP.readOnly) { return; }
 
         var button = h('button.btn.btn-primary', [
-            h('i.fa.fa-pencil'),
+            h('i', { 'class': 'fa fa-pencil', 'aria-hidden': 'true', 'aria-labelledby': Messages.profile_addDescription}),
             h('span', Messages.profile_addDescription)
         ]);
         APP.$descriptionEdit = $(button);
```

Comment:
ln 452: Please move the text `p.cp-app-profile-info` to between the "view my profile" and "share" buttons as it seems more relevant